src/frontend/js/components/PaymentButton/index.spec.tsx

```diff
@@ -183,8 +190,8 @@ describe('PaymentButton', () => {
     });
 
     // - Route to create order should have been called
-    expect(fetchMock.calls()).toHaveLength(1);
-    expect(fetchMock.lastUrl()).toBe('https://joanie.test/api/v1.0/orders/');
+    expect(fetchMock.calls()).toHaveLength(2);
```

Comment:
I feel that's it's easier to debug if we comment whitch call we're expecting as : 
```js
nbApiCall = 1 // post order
nbApiCall += 1 // submit order
expect(fetchMock.calls()).toHaveLength(nbApiCall);
```